pino.js

```diff
@@ -258,6 +259,7 @@ function pino (opts, stream) {
   // internal options
   iopts.stringify = iopts.safe ? stringifySafe : JSON.stringify
   iopts.formatOpts = {lowres: true}
+  iopts.messageKeyString = ',' + JSON.stringify('' + iopts.messageKey) + ':'
```

Comment:
No, just following the idiom of the [current line formatting](https://github.com/pinojs/pino/pull/233/files#diff-04e6fb630166e65e0231f8c63a731aa0L124) as [suggested](https://github.com/pinojs/pino/pull/233#discussion_r111674233). String templates seem like they could tighten up a few things in this code but I figured I shouldn't introduce syntax that wasn't already used elsewhere - particularly if there are Node version compatibility concerns.

Happy to change if there's interest, though!